LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaToolsTest/ConversionHelperTest.cs

```diff
@@ -20,7 +20,7 @@ public class ConversionHelperTest
         [InlineData(new byte[] { 0xF1, 0xF2, 0xF3, 0xF4 }, "F1F2F3F4")]
```

Comment:
Please revert the changes to using type instead of var for variable declarations.

Reply:
Done.